### PR TITLE
Add default community health files repository and team

### PR DESCRIPTION
**Pull request by OPC-16**
This PR is for creating a repository for default community health files such as `CODE_OF_CONDUCT.md`, `CONTRIBUTING.md`, etc.

This will help in streamlining the process of maintaining standard files, it is discussed in this [issue](https://github.com/kubevirt/community/issues/235) and for reference [here](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file) is the link to github docs.

### CODE REVIEW

File: github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml [hunk 1]
```diff
@@ -659,6 +661,13 @@ orgs:
           - aglitke
         repos:
           containerized-data-importer-api: admin
+      community-health-files:
```
Comment (dhiller) [inline github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml:664]: nit: in general we add something to distinguish teams from the repo name itself, helps in reading, here I'd suggest using maintainers.

```suggestion
      community-health-files-maintainers:
```

File: github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml [hunk 2]
```diff
@@ -659,6 +661,13 @@ orgs:
...
+        description: Default Community health files.
+        maintainers:
+          - dhiller
+          - brianmcarey
+        repos:
```
Comment (dhiller) [inline github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml:669]: We should add @aburdenthehand (our [community facilitator](https://github.com/kubevirt/community/blob/main/MAINTAINERS.md)) here also

```suggestion
          - aburdenthehand
        repos:
```

**Review by dhiller — CHANGES_REQUESTED** [review]

**Comment by dhiller** [comment]
@OPC-16 by looking at the checks section below you can find the checks that are being run when you push a commit to a PR. The `pull-kubevirt-org-github-config-updater` should tell you whether the changes you did are ok. Even if it doesn't fail, you might take a look at the log to see whether the `dry-run` would make the changes that you intended.

File: github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml (hunk 2 above)
Comment (OPC-16) [reply github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml:669]: Okay.

File: github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml (hunk 1 above)
Comment (OPC-16) [reply github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml:664]: ok, got it.

**Comment by OPC-16** [comment]
@dhiller, can you please explain the problem in [this](https://github.com/kubevirt/project-infra/pull/3074/commits/97339fa1762521aceed6e990426a2f16d7ef7f72) commit message.

**Comment by OPC-16** [comment]
Hey @dhiller, sorry didn't intend on doing [this](https://github.com/kubevirt/project-infra/pull/3074/commits/d59fbaae7665d5014d704c8e02b871ae5af547fb) commit. I amended [this](https://github.com/kubevirt/project-infra/pull/3074/commits/97339fa1762521aceed6e990426a2f16d7ef7f72) commit's message locally and tried to push it to my fork, but got ` Updates were rejected because the tip of your current branch is behind its remote counterpart.` error. So I synced my fork with the upstream with help of ui on Github but accidently also pulled commits of [main](https://github.com/kubevirt/project-infra/commits/main) branch of upstream into my feature branch. And when I pulled my feature branch locally I got this `You have divergent branches and need to specify how to reconcile them.` error, but still I tried to push this feature branch onto my fork.

Really sorry for messing this up 😟

**Comment by aburdenthehand** [comment]
@OPC-16 No worries - happens to the best of us. 
Since it is your last commit you should be able to do `git reset --hard HEAD~1` in your local. Then check your git history to ensure it is just the two commits you want to include before you push it.

**Comment by OPC-16** [comment]
Thank you @aburdenthehand, I will do the reset and try to push it back.

**Pull request closed without merging**